sdk/template/azure-sdk-template/README.md

```diff
@@ -1,94 +1,122 @@
 # README.md template
```

Comment:
Thanks for fixing this template! Please update the title too to the format expected by docsettings.

Reply:
Done, and I added the missing sections